win/inspircd_win32wrapper.h

```diff
@@ -63,6 +63,7 @@
 
 /* Disable the deprecation warnings.. it spams :P */
 #define _CRT_SECURE_NO_DEPRECATE
+#define _WINSOCK_DEPRECATED_NO_WARNINGS
```

Comment:
I looked at getting rid of the deprecated code that this #define shuts up but it requires us to use features that aren't available in all compiler versions that we support.